File: 412-fizz-buzz/412-fizz-buzz.py
class Solution:
    def fizzBuzz(self, n: int) -> List[str]:
        return [("Fizz" if i % 3 == 0 else "") + ("Buzz" if i % 5 == 0 else "") + (str(i) if i % 3 and i % 5 else "") for i in range(1, n+1)]

File: 412-fizz-buzz/412-fizz-buzz.cpp
class Solution {
public:
    vector<string> fizzBuzz(int n) {
        vector<string> res;
        for(int i = 1; i <= n; i++) {
            if(not(i % 3 or i % 5)) {
                res.push_back("FizzBuzz");
            }
            else if(not(i % 3)) {
                res.push_back("Fizz");
            }
            else if(not(i % 5)) {
                res.push_back("Buzz");
            }
            else {
                res.push_back(to_string(i));
            }
        }
        return res;
    }
};